pinit.php

```diff
@@ -43,36 +43,56 @@ function pit_admin_scripts() {
 /*=== SHORTCODES
  *==============================*/
 
+function pit_follow_shortcode( $atts ) {
+	$atts = extract( shortcode_atts( array(
+		'url' => 'http://www.pinterest.com/pinterest/',
+		'text' => 'Follow',
+	), $atts ) );
+
+	return '<a data-pin-do="buttonFollow" href="' . $url . '">' . $text . '</a>';
+}
+
 function pit_pin_shortcode( $atts ) {
 	$atts = extract( shortcode_atts( array(
 		'url' => 'http://www.pinterest.com/pin/99360735500167749/',
+		'size' => 'small',
 	), $atts ) );
 
-	return '<a data-pin-do="embedPin" href="' . $url . '"></a>';
+	if ($size == 'large' || $size == 'medium') {
+		$width = 'data-pin-width="' . $size . '" ';
+	}
+
+	else {
+		$width = '';
+	}
+	
+
+	return '<a data-pin-do="embedPin" ' . $width . 'href="' . $url . '"></a>';
 }
 
 function pit_profile_shortcode( $atts ) {
 	$atts = extract( shortcode_atts( array(
-		'url' => 'http://www.pinterest.com/pinterest/',
-		'imgWidth' => '92',
-		'boxHeight' => '175',
-		'boxWidth' => 'auto'
+		'url' => 'https://www.pinterest.com/pinterest/',
+		'imgwidth' => '80',
+		'boxheight' => '400',
+		'boxwidth' => '400',
 	), $atts ) );
 
-	return '<a data-pin-do="embedUser" href="' . $url . '" data-pin-scale-width="' . $imgWidth . '" data-pin-scale-height="' . $boxHeight . '" data-pin-board-width="' . $boxWidth . '"></a>';
+	return '<a data-pin-do="embedUser" data-pin-board-width="' . $boxwidth . '" data-pin-scale-height="' . $boxheight . '" data-pin-scale-width="' . $imgwidth . '" href="' . $url . '"></a>';
```

Comment:
I just followed your codebase and moved some value before href for personal needing but it's good to me if you want use a `sprintf()` instead! :)